web/components/answers/answer-item.tsx

```diff
@@ -68,7 +68,7 @@ export function AnswerItem(props: {
             </Row>
           </SiteLink>
           {/* TODO: Show total pool? */}
-          <div className="text-base">#{number}</div>
+          {/*<div className="text-base">#{number}</div>*/}
```

Comment:
One reason to still have this is so that the creator can prefer earlier answers when they resolve the market. Maybe we could show the number if `showChoice` is defined?

Reply:
Added it back in. I'm not sure if users understand what the number means but maybe they do!